.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+# Motivation
+
+[Include the reason behind these changes and any relevant context.]
```

Comment:
```suggestion
<!--
Include the reason behind these changes and any relevant context.]
-->
```

Reply:
This should be rendered !
The author of the PR have to read them and replace them with the appropriate text. It also helps to see if things were properly done during review

Reply:
The comment would appear in the description editor but the developer wouldn't have to delete each section's description at every PR ! The information is still there and seems handier to me.

Reply:
You'll have to be in edit mode to see comments. I think we should be able to see them at a glance for review purposes.

Reply:
In that case I think it would be preferable to render section descriptions in another way, because, so that we are able to distinguish section description from section content more easily. For example we could have : 

# Motivation
### *Include the reason behind these changes and any relevant context*

[Section content]

Reply:
What about this:
```suggestion
> **Include the reason behind these changes and any relevant context.**
```

# Motivation

> **Include the reason behind these changes and any relevant context.**

[Section content]

Reply:
Does not change anything compared to what is proposed